mariadb/charts/persistence/templates/_helpers.tpl

```diff
@@ -0,0 +1,18 @@
+{{- define "persistence.volume" -}}
+      {{- if .Values.persistence.enabled -}}
+        persistentVolumeClaim:
+          claimName: {{ template "persistence.fullname" . }}
+      {{- else -}}
+        emptyDir: {}
+      {{- end -}}
+{{- end -}}
+
+
+{{/*
+Create a default fully qualified app name.
+We truncate at 24 chars because some Kubernetes name fields are limited to this (by the DNS naming spec).
+This needs to be hardcoded as the context of `.` changes for the PVC template and the partial defined above
```

Comment:
Yeah, that could be a possibility. I guess hardcoding `.Release.Name-pvc` shouldn't be too much of an issue (especially with an optional nameOverride).

That being said, a name might be needed if the chart supported multiple PVCs, since you'd need a way to reference each specific PVC.